### ISSUE TITLE
lscgctl etc is missing from the ubuntu 24 packages

### ISSUE THREAD

**Issue by risharde** [litespeedtech/openlitespeed#431]
Hi, I noticed that performing an install via the instructions here for Ubuntu 24 https://docs.openlitespeed.org/installation/repo/ results in a working openlitespeed webserver but without files in the /usr/local/lsws/lsns/bin

Is this normal or a bug?

**Comment by risharde** [comment]
Also according to https://docs.openlitespeed.org/config/advanced/cgroups/
There is no lssetup file at location: /usr/local/lsws/lsns/bin/

Is this normal or a bug?

**Comment by dandry** [comment]
The same thing I've observed, on 22.04, both x64 and arm.
Used one-click script.

**Comment by rperper** [comment]
That is right.  The LiteSpeed Containers files are not yet available for the one-click script.  However, they are available for the precompiled code which can be obtained using:``` wget https://openlitespeed.org/packages/openlitespeed-1.7.16.tgz

**Comment by dandry** [comment]
@rperper thank you for the reply. Is there any timeline for when LiteSpeed Container files will be available in the repository and the one-click script version?

**Comment by litespeedtech** [comment]
Should be fixed in the latest 1.8.3 package now.
